src/dependency-manager/src/manager.ts

```diff
@@ -130,13 +130,31 @@ export default abstract class DependencyManager {
           internal_port = this.toInternalPort(node, interface_name);
         }
 
+        const port = external_host ? external_port : internal_port;
+        const host = external_host ? external_host : internal_host;
+
+        const internal_protocol = 'http';
+        const internal_url = internal_protocol + '://' + internal_host + ':' + internal_port;
+
+        const external_protocol = external_host ?
```

Comment:
Can you add an abstract toExternalProtocol method on the manager?
The  host will be localhost when deploying to docker kubernetes and it should be https.

Reply:
Done. Your comment about not needing changes on the api threw me off of this pattern yesterday.

Reply:
My mistake - just thought of this case